project/step06/lib/app.test.js

```diff
@@ -25,13 +25,12 @@ describe("Sfeir Schools app", () => {
 
     expect(responsePost.statusCode).toBe(201);
 
-    const responseGet = await request(app).get("/");
```

Comment:
this is the correction. Trainees have to find out how to test the GET request and find a solution to manage the _id property

Reply:
ok, I hadn't understood. I added a TODO to make it a little clearer (you can tell me if you think it's not necessary)